app/src/main/res/layout-land/profile_rename_activity.xml

```diff
@@ -73,8 +73,8 @@
           style="@style/StateButtonActive"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginTop="20dp"
-          android:layout_marginEnd="160dp"
+          android:layout_marginTop="32dp"
```

Comment:
Margin top should be `20dp` only, as the extra `12dp` is for error.
![Screenshot_1589172081](https://user-images.githubusercontent.com/9396084/81524719-dc021b80-936f-11ea-86a1-f399af90587a.png)

Reply:
@rt4914 Error message occupies its space when an error needs to be displayed. Only 2dp additional spacing is there due to wrap-content of error text which when reduced from 32dp would result in a non-multiple of 4. If I reduce the spacing by 16dp considering the error text it would look congested in screens with more than one Profileinputview hence for consistency I avoided that.

Reply:
Please mention this to @mschanteltc so as to make it clear that we want such consistency on all screens or not. I personally prefer this consistency.

Reply:
> Margin top should be `20dp` only, as the extra `12dp` is for error.
> ![Screenshot_1589172081](https://user-images.githubusercontent.com/9396084/81524719-dc021b80-936f-11ea-86a1-f399af90587a.png)

@mschanteltc PTAL

Reply:
Hi Nikita,

The label "New Name" and the error message "Names can only have..." should be slightly indented, about 8 px from the start of the textbox. The color of the textbox border and error message is now #923026 (it was previously orange but did not pass contrast checker). I'm in the process of updating these in the mocks and will update this issue when completed.

Reply:
I updated the [landscape](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/) and [portrait](https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/) mocks so that the textbox inputs are consistent. 

All textboxes with a label (this **may be different** from Exploration Player's answer input) have space beneath it reserved for an [error message](https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/62215bbd-1f4e-448e-b370-19c4a06fa814/PC-SP-AD-PINs-Match-Error-). This error message is invisible and appears when a user inputs something that doesn't follow guidelines. By doing this, we don't have to push the content beneath to make room for the error message. 

This change affects the spacing of the textboxes relative to other objects, but the formatting remains consistent. Notice in this [mock](https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/9603eda8-8d20-4377-9458-d5deec58b4d9/PC-NP-Admin-Settings-Dialog-Empty-) that there is no error message, but tapping underneath the text input shows an invisible box reserved for the error.